package.json

```diff
@@ -44,7 +44,8 @@
   "homepage": "https://angular-maps.com",
   "peerDependencies": {
     "@angular/common": "^4.0.0 || ^2.0.0",
-    "@angular/core": "^4.0.0 || ^2.0.0"
+    "@angular/core": "^4.0.0 || ^2.0.0",
+    "js-marker-clusterer": "^1.0.0"
```

Comment:
We need to have a separate @agm package to handle the different dependencies. (See my last comment)